fuzzers/afl/fuzzer.py

```diff
@@ -91,13 +91,17 @@ def run_afl_fuzz(input_corpus,
     ]
     if additional_flags:
         command.extend(additional_flags)
+    dictionary_path = utils.get_dictionary_path(target_binary)
```

Comment:
Do you think it's better to make users get their own dictionary or should we pass it as an argument?
@lszekeres WDYT?

Reply:
I thought about this. dictionary should always be derivable from target_binary, that is why i went with this simpler model. Also, new fuzzers might not even use this feature, so thought to keep api/arguments simple.

Reply:
Not all new fuzzers will support this and this attribute is derived from target_binary, so thought to create a helper function and keep api simpler.

Reply:
SGTM

Reply:
I'm still thinking about this. Passing arguments explicitly (even if there are more of them) is IMO easier to deal with than using helper functions to deal with implicit behavior.

I'm trying to ensure that these files never get as complicated as the ClusterFuzz launchers.